mne/beamformer/tests/test_lcmv.py

```diff
@@ -697,6 +697,74 @@ def test_lcmv_ctf_comp():
         make_lcmv(info_comp, fwd, data_cov)
 
 
+@testing.requires_testing_data
+@pytest.mark.parametrize('pick_ori', ['max-power', 'normal', 'vector'])
+@pytest.mark.parametrize('reg', (0.05, 0.))
```

Comment:
The formula I implemented here does definitely not hold for vector beamformers ....

Reply:
I'll just kill `vector` for now then